common/azure-common-test/src/main/java/com/azure/common/test/http/RecordClient.java

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This class exists already: $\common\azure-common-test\src\main\java\com\azure\common\test\policy\RecordNetworkCallPolicy.java

Reply:
One question here, why it has two different way of handling the recorded network, (e.g. PlaybackClient in playback mode and RecordNetworkCallPolicy in recoreded mode)?

Reply:
PlaybackClient is when we don't have a live connection to an Azure Service. So we read a session-record from the test/resources/session-records/*.json location. And for each network call, play back a previously copied response.
RecordNetworkCallPolicy is for when we have a connection to an Azure Service. We save each HTTP request and response so that we can verify its results or play it back later.

Reply:
Thanks for the explanation. Really helpful.